tests/src/test/java/io/streamnative/pulsar/handlers/kop/KopProtocolHandlerTestBase.java

```diff
@@ -329,24 +335,28 @@ protected final void internalCleanup() throws Exception {
             if (pulsar != null) {
                 stopBroker();
             }
-            if (mockBookKeeper != null) {
-                mockBookKeeper.reallyShutdown();
-            }
-            if (mockZooKeeper != null) {
-                mockZooKeeper.shutdown();
-            }
             if (sameThreadOrderedSafeExecutor != null) {
                 sameThreadOrderedSafeExecutor.shutdown();
             }
-            if (bkExecutor != null) {
-                bkExecutor.shutdown();
-            }
+            cleanupStorage();
```

Comment:
Why did we `cleanupStorage()` here ?  I think your original purpose is split broker and storage cleanups right?

Reply:
It's right. I'll remove this line and try to verify the old data still exists after restart.